serenata_toolbox/federal_senate/federal_senate_dataset.py

```diff
@@ -0,0 +1,83 @@
+import os.path
+from urllib.request import urlretrieve
+import numpy as np
+import pandas as pd
+
+class FederalSenateDataset:
+    URL = 'http://www.senado.gov.br/transparencia/LAI/verba/{}.csv'
+
+    def __init__(self, path):
+        self.path = path
+
+    def fetch(self):
+        urls = [self.URL.format(year) for year in range(2008, 2018)]
```

Comment:
This hardcoded `2018` seems like a problem. We could use something like that to avoid bumping this year every new years eve:

```python
from datetime import date

…

    def fetch(self):
        next_year = date.today().year + 1
        urls = [self.URL.format(year) for year in range(2008, next_year)]
        …
```